json-el/src/main/java/io/zeebe/msgpack/el/JsonConditionInterpreter.java

```diff
@@ -35,35 +35,48 @@
 
   private final JsonPathCache cache = new JsonPathCache();
 
+  public boolean evalStrict(final JsonCondition condition, final DirectBuffer json) {
```

Comment:
The point of this PR is to provide non-strict version, so there would be no incident on accessing missing property, but just a null value, as it is in JS. This would also fail the comparison, because null is only equals to null and nothing else, but would also allow for comparing with null, in turn allowing to make a BPM decision on what to do on null or missing property.

However, if default strict mode is meaningful (i don't really see where), i think it would be best to add it as zeebe-wide configuration option. And probably as xml-attribute for specific condition too.

> Can you please also update the documentation so that it is clear how to deal with null or missing properties?

This PR makes it possible to deal seamlessly -- if any property on selector path is missing, it is evaluating whole selector to null and allowing to test on that and act accordingly.

Reply:
Removed strict version altogether. I think if we ever would want it again, `in` operator described in https://github.com/zeebe-io/zeebe/issues/1751#issuecomment-447519203 would do better job for the same scenarios.